src/kindlemint/payments/stripe_checkout.py

```
@@ -0,0 +1,407 @@
+        return
+    
+    # Create test checkout session
+    result = checkout.create_checkout_session(
+        product_name="Test - Large Print Sudoku Masters Vol 1",
+        price_cents=899,
+        customer_email="test@example.com",
+        metadata={'test': True}
+    )
+    
+    if result['success']:
+        print(f"✅ Checkout session created!")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)